builder/chroots.go

```diff
@@ -80,6 +82,416 @@ func readBuildChrootsConfig(path string) (*buildChrootsConfig, error) {
 	return cfg, nil
 }
 
+func updateBundlePackages(set bundleSet, packages *sync.Map) {
+	for _, b := range set {
+		for k := range b.DirectPackages {
+			deps, ok := packages.Load(k)
+			if !ok {
+				continue
+			}
+
+			for _, d := range deps.([]string) {
+				d = strings.Trim(d, " ")
+				if len(d) > 0 {
+					b.DirectPackages[d] = true
+				}
+			}
+		}
+	}
+}
+
+func isBannedPath(path string) bool {
+	if path == "/" {
+		return true
+	}
+
+	bannedPaths := []string{
+		"/var/lib/",
+		"/var/cache/",
+		"/var/log/",
+		"/dev/",
+		"/run/",
+		"/tmp/",
+	}
+
+	for _, p := range bannedPaths {
+		if strings.HasPrefix(path, p) {
+			return true
+		}
+	}
+	return false
+}
+
+// Clear Linux OS ships several symlinks in the filesystem package, and RPM
+// packages may install files to paths with one of these symlinks as a path
+// prefix. Short term, resolve file paths using the symlinks from the
+// filesystem. Long term, we need to check every path component and resolve.
+func resolveFileName(path string) string {
+	re := regexp.MustCompile(`^(/bin/)`)
+	path = re.ReplaceAllLiteralString(path, "/usr/bin/")
+	if strings.HasPrefix(path, "/usr/bin/") {
+		return path
+	}
+	re = regexp.MustCompile(`^(/usr/sbin/)`)
+	path = re.ReplaceAllLiteralString(path, "/usr/bin/")
+	if strings.HasPrefix(path, "/usr/bin/") {
+		return path
+	}
+	re = regexp.MustCompile(`^(/sbin/)`)
+	path = re.ReplaceAllLiteralString(path, "/usr/bin/")
+	if strings.HasPrefix(path, "/usr/bin/") {
+		return path
+	}
+	re = regexp.MustCompile(`^(/lib64/)`)
+	path = re.ReplaceAllLiteralString(path, "/usr/lib64/")
+	if strings.HasPrefix(path, "/usr/lib64/") {
+		return path
+	}
+	re = regexp.MustCompile(`^(/lib/)`)
+	return re.ReplaceAllLiteralString(path, "/usr/lib64/")
+}
+
+func addFileAndPath(destination map[string]bool, absPathsToFiles ...string) {
+	for _, file := range absPathsToFiles {
+		if isBannedPath(file) {
```

Comment:
Is there any chance we want to know (i.e., output/log of some kind) when this case hits? (i.e., would the user want to know that a file is being skipped?)

Reply:
No, it would really spam if we reported every time we hit a banned path. I don't think we need to tell the user some things are getting skipped that have always been skipped by swupd.